assets/js/script.js

```diff
@@ -0,0 +1,181 @@
+$(function () {
+  // Create P1 & P2 divs & give them a color.
+  var $player1 = $('<div class="player1 lupid">')
+  var $player2 = $('<div class="player2 pig">')
+
+  var currentPlayer = $player1
+
+  var totalStepsTakenByP1 = 1 // track where player is supposed to be at.
+  var totalStepsTakenByP2 = 1
+
+  var $circle = $('.circle') // circle img to indicate which player's turn
+
+    // Create dice event
+  var $dieButton = $('.dieButton')
+  var $dieValue = $('.dieValue')
+  $dieButton.on('click', rollDice)
+
+
+  // create autoPlay event
+  var $autoPlayButton = $('.auto-play')
+  $autoPlayButton.on('click', autoPlayButton)
+  var clear;
+
+  // Create reset event
+  var $resetButton = $('.resetButton')
+  $resetButton.on('click', resetButton)
+
+  // Create audio event
+  var $soundButton = $('.sound')
+  $soundButton.on('click' , playAudio)
+
+  function playAudio () {
+    document.getElementsByClassName('audio')[0].play()
+  }
+
+  function startPosition () {
+    $('#1').append($player1) // grid 1
+    $('#1').append($player2) // grid 1
+  }
+
+  function rollDice () {
+    var randomDiceResult = 1 + Math.floor(Math.random() * 6)
+
+    $dieValue.text(`${randomDiceResult}`)
+
+    if (currentPlayer === $player1) {
+      totalStepsTakenByP1 += randomDiceResult
+      $player1.detach().appendTo(`#${totalStepsTakenByP1}`) // move player 1 + dice value
+      $circle.css("left", "147px") // indicate player1 turn
+      painOrPleasureP1()
+
+      if (randomDiceResult === 6) {
+        currentPlayer = $player1 // if dice is 6 go again
+        $circle.css("left", "25px")
+      } else {
+        currentPlayer = $player2
+        $circle.css("left", "147px")
+      }
+    } else if (currentPlayer === $player2) {
+      totalStepsTakenByP2 += randomDiceResult
+      $player2.detach().appendTo(`#${totalStepsTakenByP2}`) // move player 1 + dice value
+      painOrPleasureP2()
+      $circle.css("left", "25px") // indicate player2 turn
+
+      if (randomDiceResult === 6) {
```

Comment:
You may try this later : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator